pyproject.toml

```diff
@@ -22,6 +22,8 @@ tavily-python = "^0.3.0"
 tabulate = "^0.9.0"
 pysqlite3-binary = {version="^0.5.2.post3", optional=true}
 langchain-community = "^0.0.16"
+tenacity = "^8.2.3"
+joblib = "^1.3.2"
```

Comment:
I guess you should have an updated poetry.lock now too

Reply:
I should, and I did 😱 Some mishmash while splitting the PRs. Fixed!